Mention the update to BIRT 4.20.0. (#5070)
centreon/centreon-documentation: versioned_docs/version-26.10/reporting/report-development.md

title: Creating custom reports
---

Compatibility: BIRT 4.20.0+

> > Upgrading to BIRT 4.20.0 may result in changes that affect your custom templates. We recommend that you check them after the migration.

Reviewer: ```suggestion
> Upgrading to BIRT 4.20.0 may result in changes that affect your custom templates. We recommend that you check them after upgrading from an earlier BIRT version.
```
